src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
Ok, if that's the case then the count of selectors for `@page` should be 1, not 0.